modules/nf-core/gatk4/markduplicates/gatk4-markduplicates.diff

```diff
@@ -1,45 +0,0 @@
-Changes in module 'nf-core/gatk4/markduplicates'
---- modules/nf-core/gatk4/markduplicates/main.nf
-+++ modules/nf-core/gatk4/markduplicates/main.nf
-@@ -2,10 +2,10 @@
-     tag "$meta.id"
-     label 'process_medium'
- 
--    conda "bioconda::gatk4=4.3.0.0"
-+    conda "bioconda::gatk4=4.3.0.0 bioconda::samtools=1.16.1"
-     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
--        'https://depot.galaxyproject.org/singularity/gatk4:4.3.0.0--py36hdfd78af_0':
--        'quay.io/biocontainers/gatk4:4.3.0.0--py36hdfd78af_0' }"
-+        'https://depot.galaxyproject.org/singularity/mulled-v2-d9e7bad0f7fbc8f4458d5c3ab7ffaaf0235b59fb:551156018e5580fb94d44632dfafbc9c27005a0e-0':
-+        'quay.io/biocontainers/mulled-v2-d9e7bad0f7fbc8f4458d5c3ab7ffaaf0235b59fb:551156018e5580fb94d44632dfafbc9c27005a0e-0' }"
```

Comment:
we should still used the updated version I guess, but need a new mulled container 😬

Reply:
This is what we do, this whole file has been removed, we no longer patch that